source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Tests/Domain/Charges/ChargeTests.cs

```diff
@@ -531,6 +534,42 @@ public void UpdatePrices_WhenNoPointsExistBetweenStartAndStopDate_PointsUpdated(
             actualPriceSecondAdded.Price.Should().Be(7.00m);
         }
 
+        [Fact]
+        public void UpdatePrices_WhenChargeIsTaxTariff_AndSenderIsNotSystemOperator_PointsNotUpdated()
+        {
+            // Arrange
+            var points = new List<Point>
+            {
+                new(1.00m, InstantHelper.GetTodayPlusDaysAtMidnightUtc(0)),
+                new(2.00m, InstantHelper.GetTodayPlusDaysAtMidnightUtc(1)),
+                new(3.00m, InstantHelper.GetTodayPlusDaysAtMidnightUtc(2)),
+                new(4.00m, InstantHelper.GetTodayPlusDaysAtMidnightUtc(3)),
+                new(5.00m, InstantHelper.GetTodayPlusDaysAtMidnightUtc(4)),
+            };
```

Comment:
Could we do fine with just one existing point?
```suggestion
            var points = new List<Point> { new(1.00m, InstantHelper.GetTodayPlusDaysAtMidnightUtc(0)) };
```

Reply:
Yes that would be fine. I have changed it to only one.